templates/zerver/keyboard_shortcuts.html

```diff
@@ -155,6 +155,10 @@
           <td class="hotkey">M</td>
           <td class="definition">{% trans %}Toggle topic mute{% endtrans %}</td>
         </tr>
+        <tr>
+          <td class="hotkey">O</td>
+          <td class="definition">{% trans %}Toggle condense message view{% endtrans %}</td>
```

Comment:
I think the description should be something more user-friendly like "Collapse/uncollapse selected message", since that's what we call it in the message chevron menu.  Can you change that here and above?

Otherwise this PR looks great!